marker-api.js

```diff
@@ -1,780 +1,780 @@
-// Each function returns data to be 
-// parsed to json and sent to requestor
-
-var models = require('./models/marker.js'),
-    Marker = models.Marker,
-    DeletedMarker = models.DeletedMarker,
-    _ = require("underscore"),
-    fs = require('fs'),
-    formidable = require('formidable'),
-    crypto = require('crypto'),
-    Q = require('q'),
-	mongoose = require('mongoose'),
-	ObjectId = mongoose.Types.ObjectId,
-    moment = require('moment');
-
-
-// Private functions
-
-// This function uses formidable to
-// parse multipart marker posts, ensure that
-// they are unique and return their
-// values if so.
-// @returns Q promise object
-function parse_marker_post (req, res) {
-    var form,
-        fields,
-        file,
-        image_hash,
-        form_load;
-
-    console.log('parse marker post');
-    // Parse form data
-    var form = new formidable.IncomingForm();
-    form.uploadDir = __dirname + '/public/photos/';
-    form.hash = 'md5';
-
-    // Wait for load to finish
-    form_load = Q.defer();
-
-    // Parse all non-file form data then resolve
-    form.parse(req, function (err, fields, files) {
-
-        if (err) {
-            form_load.reject(err);
-            return;
+  // Each function returns data to be 
+  // parsed to json and sent to requestor
+
+  var models = require('./models/marker.js'),
+      Marker = models.Marker,
+      DeletedMarker = models.DeletedMarker,
+      _ = require("underscore"),
+      fs = require('fs'),
+      formidable = require('formidable'),
+      crypto = require('crypto'),
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/charlieblevins/dukweb/pullRequest?prid=9134160)
```suggestion
      crypto = require("crypto"),
```